BUILD.md

```diff
@@ -111,6 +114,22 @@ Step by step procedure:
    * Compile the whole project by selecting the `Build` -> `Build Solution`
      menu entry.
 
+ * On Windows (cmake):
+   * Change into you `Freeorion` directory.
+   * Create a `build` directory, which will contain all compile FreeOrion
+     build artifacs.
+   * Change into the `build` directory on the command line.
+   * Execute cmake to generate Makefiles:
+
+     ```
+     cmake .. -G "Visual Studio 15 2017"
+     ```
+   * Compile the whole project by calling `MSBuild.exe -p:Configuration=Release`
```

Comment:
I build a fresh SDK and then the project files, which went fine. But when I tried this line, I get:
````
C:\Users\Geoff\Desktop\FreeOrionSDK_10\FreeOrion\build>MSBuild.exe -p:Configuration=Release
Microsoft (R) Build Engine version 14.0.25420.1
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains
more than one project or solution file.

C:\Users\Geoff\Desktop\FreeOrionSDK_10\FreeOrion\build>
````

Reply:
This is my fault, it should be `MSBuild.exe -p:Configuration=Release FreeOrion.sln`